avocado/plugins/docker.py

```diff
@@ -173,6 +174,10 @@ def configure(self, parser):
                                 "docker' or other base docker options like "
                                 "hypervisor. Default: '%(default)s'",
                                 metavar="CMD")
+        cmd_parser.add_argument("--docker-options", default="",
```

Comment:
I don't think we should default assign empty str objects to arguments. But this is not the first occurrence in Avocado, so no need to change it.